_posts/2024-06-06-strimzicon2024-roundup.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title:  "StrimziCon 2024: Event Roundup and Session Recordings"
+date: 2024-06-06
+author: kate_stanley
+---
+
+StrimziCon 2024 is finished!
+
+![StrimziCon 2024 Banner](/assets/images/posts/2024-01-29-strimzicon2024-banner.png)
+
+### StrimziCon Roundup
+
+In the end we had:
+* 8 sessions
+* 11 speakers from 6 organizations
+* Over 5 hours of content
+
+A massive thank you to all the attendees (around 200!) who joined us live on the day, it was so great to see how many people are interested in Strimzi.
+
+We also of course need to thank everyone who helped make StrimziCon a reality - the speakers, program committee and of course CNCF who provided so much assistance in running the event.
+
```

Comment:
Maybe adding even just a couple of lines around ... "we had sessions around Strimzi internals to understand how it works at the core, as well as use cases and how companies are using Strimzi in production today" ... again I will leave you the best wording :-)

Reply:
I've added some more words